components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -7134,7 +7134,7 @@ def getCall(self):
             "${getCallable}"
             "rooted!(in(cx) let rootedThis = ${thisObj});\n"
             "let ok = ${callGuard}JS_CallFunctionValue(\n"
-            "    cx, rootedThis.handle(), callable.handle(),\n"
+            "    *cx, rootedThis.handle(), callable.handle(),\n"
```

Comment:
I suspect this is going to cause problems in the short term.

Reply:
Don't `JS_*` functions expect `*mut JSContext` of js::jsapi? By dereferencing, I think we provide them with the type they want. Or did I miss something?